components/textbox/src/_mixin.scss

```diff
@@ -271,18 +271,33 @@ $utrecht-support-prince-xml: false !default;
   }
 }
 
+@mixin utrecht-textbox--aantalmarkten-size {
+  /* Nummer onder 9999 */
+  --utrecht-textbox-value-max-length: 10;
+}
+
```

Comment:
`aantalmarkten` hoeven we niet als herbruikbare mixin te hebben, dat lijkt me niet algemeen nuttig.

Trouwens: als het nummer onder 9999 is, waarom is `max-length` dan niet 4?

Reply:
Er zijn best veel velden die ik bewust nog niet in de mixin heb gedaan, omdat ik ze niet nuttig vond. Maar nu weet ik dat het aantalmarkten ook een van deze velden is. De 'max length' had ik gewoon op het totaal gezet, maar het zou wel indd 4 zijn anders.

`--utrecht-textbox-value-max-length: 4;`

<img width="300" alt="Screenshot 2024-11-26 at 16 24 54" src="https://github.com/user-attachments/assets/5aaa1b0e-051d-41b0-bdc1-f3f1ac767065">